accelerator/abstract_accelerator.py

```diff
@@ -222,7 +222,11 @@ def LongTensor(self):
         ...
 
     @abc.abstractmethod
-    def pin_memory(self, tensor):
+    def pin_memory(self, tensor, align=False):
```

Comment:
After further thought, I think the second API is better because it allows the clients to use other alignments besides the default choice of the accelerator.